test/parser_test.cc

```diff
@@ -4,6 +4,19 @@
 
 #include <vector>
 
+TEST(ParseTest, Parse_PraseStringCorrectly) {
```

Comment:
Can we also add a unit test for `utils::Split` since we update it's implementation.